runtime/doc/eval.txt

```diff
@@ -5138,6 +5138,22 @@ msgpackparse({list})				   {Nvim} *msgpackparse()*
 			representing extension type. Second is 
 			|readfile()|-style list of strings.
 
+msgpane({message}[, {highlight}[, {open}]])		{Nvim} *msgpane()*
```

Comment:
So is this a completely separate mechanism from `:messages`? I thought the msgpane would collect all `:echomsg` messages.